src/Appwrite/Messaging/Adapter/Realtime.php

```diff
@@ -129,7 +138,7 @@ public function hasSubscriber(string $projectId, string $role, string $channel =
      * @param array $options
      * @return void
      */
-    public static function send(string $projectId, array $payload, array $events, array $channels, array $roles, array $options = []): void
+    public function send(string $projectId, array $payload, array $events, array $channels, array $roles, array $options = []): void
```

Comment:
we have static usages of this function in a couple of places. 
https://github.com/search?q=repo%3Aappwrite%2Fappwrite%20%3A%3ASend(&type=code

So not sure this will work as expected